cdk/infrastructure/constructs/dashboards/backend.py

```diff
@@ -0,0 +1,211 @@
+        )
+        response_length_widget = GraphWidget(
+            left=[response_length_metric],
+            left_y_axis=YAxisProps(show_units=False),
+            period=cdk.Duration.seconds(60),
+        )
+        self._widgets.append(response_length_widget)
+
+    def _define_response_2xx_count_widget(self) -> None:
+        response_2xx_count_metric_filter = MetricFilter(
+            self,
+            '2xxResponseCountFilter',
+            log_group=self.props.log_group,
+            metric_namespace=self.props.config.branch,
+            metric_name='2xx Response Count',
+            filter_pattern=FilterPattern.string_value('$.status', '=', '2**'),
+            metric_value='1',
+        )
+        response_2xx_metric = response_2xx_count_metric_filter.metric(
+            label='2xx Response Count',
+            statistic=Stats.SUM,
+        )
+        response_2xx_widget = GraphWidget(
+            left=[response_2xx_metric],
+            left_y_axis=YAxisProps(show_units=False),
+            period=cdk.Duration.seconds(60),
+        )
+        self._widgets.append(response_2xx_widget)
+
+    def _define_response_4xx_count_widget(self) -> None:
+        response_4xx_count_metric_filter = MetricFilter(
+            self,
+            '4xxResponseCountFilter',
+            log_group=self.props.log_group,
+            metric_namespace=self.props.config.branch,
+            metric_name='4xx Response Count',
+            filter_pattern=FilterPattern.string_value('$.status', '=', '4**'),
+            metric_value='1',
+        )
+        response_4xx_metric = response_4xx_count_metric_filter.metric(
+            label='4xx Response Count',
+            statistic=Stats.SUM,
+        )
+        response_4xx_widget = GraphWidget(
+            left=[response_4xx_metric],
+            left_y_axis=YAxisProps(show_units=False),
+            period=cdk.Duration.seconds(60),
+        )
+        self._widgets.append(response_4xx_widget)
+
+    def _define_response_5xx_count_widget(self) -> None:
+        response_5xx_count_metric_filter = MetricFilter(
+            self,
+            '5xxResponseCountFilter',
+            log_group=self.props.log_group,
+            metric_namespace=self.props.config.branch,
+            metric_name='5xx Response Count',
+            filter_pattern=FilterPattern.string_value('$.status', '=', '5**'),
+            metric_value='1',
+        )
+        response_5xx_metric = response_5xx_count_metric_filter.metric(
+            label='5xx Response Count',
+            statistic=Stats.SUM,
+        )
+        response_5xx_widget = GraphWidget(
+            left=[response_5xx_metric],
+            left_y_axis=YAxisProps(show_units=False),
+            period=cdk.Duration.seconds(60),
+        )
+        self._widgets.append(response_5xx_widget)
+
+    def _define_dashboard(self) -> None:
+        self.dashboard = Dashboard(
+            self,
+            'DashBoard'
+        )
```

Comment:
I think we might need to define `dashboard_name` (https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_cloudwatch/Dashboard.html) here so we can tell which backend it's reporting on. Something like `dashboard_name = f'{self.props.config.common.project_name}-{self.props.config.branch}-backend'`.